merlin/models/tf/transformers/transforms.py

```diff
@@ -193,6 +193,10 @@ def __init__(self, summary: str, initializer_range: float = 0.02, **kwargs):
         config = SimpleNamespace(summary_type=summary)
         super().__init__(config, initializer_range=initializer_range, **kwargs)
 
+    def get_config(self):
+        config = super().get_config()
```

Comment:
we may also need to also save the `summary` argument from the `__init__` in this config, and implement `from_config` to restore the layer to the same state correctly

Reply:
You're right, I have updated the `SequenceSummary` to make it more flexible. 
The reason I didn't add `from_config` is that `SequenceSummary` was used as a base class of the transform blocks `SequenceLast`, `SequenceFirst`, `SequenceMean`, and `SequenceClsIndex` where the `summary` parameter was set to a default value for each block.